website/docs/docs/building-a-dbt-project/metrics.md

```diff
@@ -65,39 +73,37 @@ metrics:
+### Metrics properties
 
-**See and select dependencies.** As with exposures, it's possible to see everything that rolls up into a metric (`dbt ls -s +metric:*`), and visualize them in [dbt documentation](documentation).
+You can see how these properties are used in the [example YAML file](#declaring-a-metric).
 
-<Lightbox src="/img/docs/building-a-dbt-project/dag-metrics.png" title="Metrics appear as pink nodes in the DAG (for now)"/>
+| Field       | Description                                               | Example                           | Required? |
+|-------------|-----------------------------------------------------------|-----------------------------------|-----------|
+| name        | Unique identifier for the metric                          | new_customers                     | Yes       |
+| model       | The dbt model that powers this metric                     | dim_customers                     | Yes       |
+| label       | A short name / label for the metric                       | New Customers                     | No        |
+| description | Long form, human-readable description for the metric      | "The number of paid customers..." | No        |
+| type        | Type of calculation to perform when evaluating a metric   | count_distinct                    | Yes       |
+| sql         | Expression to aggregate or calculate over                 | user_id                           | Yes       |
+| timestamp   | Time-based component of the metric                        | signup_date                       | Yes       |
+| time_grains | One or more "grains" at which the metric can be evaluated | [day, week, month]                | Yes       |
+| dimensions  | List of dimensions to group or filter the metric by       | plan <br/> country                | No        |
+| filters     | List of filters to apply before calculating the metric    | field: is_paying<br/>value: true  | No        |
+| meta        | Arbitrary key/value store                                 | {team: Finance}                   | No        |
 
-### Open questions
+## Open questions
 
-- Should metrics be defined on top of more strongly typed **attributes**, rather than columns? [dbt-core#4090](https://github.com/dbt-labs/dbt-core/issues/4090)
+- Should we define metrics on top of more strongly typed **attributes**, rather than columns? [dbt-core#4090](https://github.com/dbt-labs/dbt-core/issues/4090)
 - Should metrics include support for joins? How should dbt know about foreign-key relationships between models? [dbt-core#4125](https://github.com/dbt-labs/dbt-core/issues/4125)
 - Should metrics inherit configurations from the models on which they are defined? Should it be possible to define metrics directly on models/columns, like tests?
 
 These are just a start! We welcome you to check out open issues on GitHub, and join the conversation.
+
+## Further reading
+
+* [`metric:` selection method](node-selection/methods#the-metric-method)
```

Comment:
We have a tendency to include links to "related documentation" near the top of each page, in case the user is looking for something else related to the same subject matter. There aren't many docs for metrics yet, though. I think it might make more sense to stick this particular link under `**See and select dependencies**` above.

Reply:
Done!